test/unit/test_sap_cli_atc.py

```
@@ -495,6 +501,15 @@ def test_priority_filter_2(self):
 ''')
         self.assertEqual(1, ret)
 
+    def test_priority_filter_0_skips_object(self):
+        output = StringIO()
+        ret = sap.cli.atc.print_worklists_as_html_to_stream([self.worklist], output, priority_filter=0)
+        self.assertEqual(output.getvalue(),
+'''<table>
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)